.github/workflows/build-image.yaml

```diff
@@ -5,10 +5,10 @@ on:
     inputs:
       ref:
         type: string
-        default: ${{ github.ref }}
+        default: main
       repository:
         type: string
-        default: ${{ github.repository }}
+        default: gingersnap-project/db-syncer
```

Comment:
The defaults here still need to be `${{ github.repository }}` and `${{ github.repository }}` so that calls to this workflow from `publish-dev-images.yaml` work as expected.

When we call from `pubish-release.yaml` -> `publish-image.yaml` -> `build-image.yaml` we always explicitly set these fields, so the defaults are never used.